lib/decompress_lz77.ml

```diff
@@ -115,7 +115,7 @@ let deflate ?(max_fardistance = (1 lsl 15) - 1) src t =
   in
   let find_match idx =
     let max a b =
-      match a, b with
+      match (a, b) with
```

Comment:
This seems to have snuck in because this PR is rebased on top of another recent change, I left a comment there instead. Please ignore @TheFrenchTroll :)